src/abstracts/SablierV2Base.sol

```diff
@@ -22,18 +22,14 @@ abstract contract SablierV2Base is
     using SafeERC20 for IERC20;
 
     /*//////////////////////////////////////////////////////////////////////////
-                                  PUBLIC CONSTANTS
+                                  STATE VARIABLES
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @inheritdoc ISablierV2Base
```

Comment:
I think we should keep this order:
1. Constants and immutable
2. Mutable variables

The order of mutable variables plays a key role in storage layout so we should keep them together. Technically speaking, inserting constants in between mutable variables does not have any impact as long as the order of mutable variables is optimized for storage. But from a readability point of view, I think having immutable variables followed by mutable variables looks good.

Reply:
> But from a readability point of view,  I think having immutable variables followed by mutable variables looks good.

I agree.